### ISSUE TITLE
Add a LICENSE file to this repository

### ISSUE THREAD

**Issue by gaurav** [cancerDHC/ccdh-terminology-service#31]
This repository should probably have a LICENSE file so its licensing is clear.

**Comment by joeflack4** [comment]
Good idea. @jiaola Bumping this to you; I'm not sure which kind of license is appropriate for the work done under this grant.

**Comment by gaurav** [comment]
@monicacecilia just let me know that we should "use the BSD Open Source Software License - specifically the BSD3 - Clause License on the Open Source Initiative Website, found at http://opensource.org/licenses/bsd-3-clause".
